### ISSUE TITLE
markdown -> pdf: error; math in \href description fails, latex writer bug?

### ISSUE THREAD

**Issue by ViktorQvarfordt** [jgm/pandoc#583]
Reproducing:

```
me@host:~$ echo '..text.. [$\LaTeX$](http://example.org) ..text..' | pandoc -f markdown -o test.pdf
pandoc: Error producing PDF from TeX source.
! You can't use `\spacefactor' in math mode.
\@->\spacefactor 
                 \@m 
l.35 ..text.. \href{http://example.org}{$\LaTeX$}
```

This does with html writer, so I assume this is a bug in the LaTeX writer.

**Comment by v1kn** [comment]
@jgm
I frequently convert my md notes into both html and pdf files, the comment You
made directly above really suits my usecase (I'd like to be able to use `\LaTeX`
in both formats.

However, I don't know how I should do it, the link in Your comment seems broken.
Could You please repeat the correct way of configuring pandoc to display the latex
command in both html and pdf at the same time?

Furthermore, below command that has `\LaTeX` in `test.md` works.

    pandoc -f markdown test.md -o test.pdf

However, if I change `markdown` to `markdown_github`, the command stops working.
Unfortunately I always use the github variant when writing my md files. 

I used [this section on markdown variants][1] from pandoc documentation, and I subtracted all the extensions from the variant, like so:

    pandoc -f markdown_github-pipe_tables-raw_html-etc

But, even when I substract all the extensions from `markdown_github`, the command still does not work, treating `\LaTeX` verbatim

So, in summary,
I'd like to ask whether there is a way for me, to be able to use `\LaTeX` in my md files, that will be properly converted into both html and pdf, and while using `markdown_github` as an input format

**Comment by suknat** [comment]
@jgm There does seem to be a problem with your post of Aug 13 2012. Any chance you could repost this?

I tried using `pandoc-tex2svg` to solve this but that simply produced a PDF with no math! Oddly (or perhaps not so oddly to those who understand these things better) no error message was produced.

**Comment by jgm** [comment]
I updated the comment.

**Comment by suknat** [comment]
Thanks @jgm that works, in principle. In practice, the image now has to be scaled to some target size so it can appear inline. Can't do html width/height adjustment as those cannot be made relative to the size of the text AFAIK. My output formats are PDF or epub, hence the need to worry about multiple font sizes.
